jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/startup/CodeWhispererProjectStartupActivity.kt

```diff
@@ -41,7 +41,7 @@ class CodeWhispererProjectStartupActivity : StartupActivity.DumbAware {
         if (runOnce) return
 
         // Reconnect CodeWhisperer on startup
-        promptReAuth(project)
+        promptReAuth(project, true)
```

Comment:
nit: Can you add the parameter name next to the boolean value?

Reply:
This is fixed in latest commit. Thanks.